### ISSUE TITLE
Question > Is this widget bundled with KDE Plasma code?

### ISSUE THREAD

**Issue by Musikolo** [dfaust/plasma-applet-netspeed-widget#18]
Hi,

I wonder if this widget now part of official code of KDE Plasma and it comes bundled with it. I'm asking because I found it is available in machine, but I don't have it installed as an external dependency.

Thank you.

**Comment by dfaust** [comment]
No it's not part of Plasma, but maybe your distribution installed it by default. Try searching for "netspeed" in your package manager. What distribution do you use?

**Comment by Musikolo** [comment]
@dfaust, I have Arch Linux, and the thing is that I already searched in my package manager (`pacman`). However, further looking into, I found out the widget was available under `~/.local/share/plasma/plasmoids/org.kde.netspeedWidget`. It's very weird, because when I looked at the `Netspeed Widget` in the `Download New Plasma Widget` window, it showed as uninstalled. The installation happened long time ago (I didn't remember about it), so there might be some sort of disconnect here. Anyway, now that I found it, I managed to remove it. All I wanted is to get it installed as a regular package from [AUR](https://aur.archlinux.org/packages/plasma5-applets-netspeed/). 

Thanks a lot for your help, and congrats for your AwEsOmE plasmoid!!
